tests/common/plugins/conditional_mark/README.md

```diff
@@ -159,3 +159,12 @@ A new pytest command line option is added for specifying location of the conditi
 The plugin is open for extension in couple of areas:
 * Collect more facts. Then more variables can be used in condition string for evaluation.
 * Add more arguments for marks, not just the current `reason` argument.
+
+## How to use pre-commit hooks to check the order in the yaml files
+We keep the test cases in alphabetical order in the yaml files, and we use pre-commit hook to check if the test cases in correct order.
+In order to use git hook, we should specify the hook folder using`git config core.hooksPath 'path'` before commit.
+
+Assume we use git under the `sonic-mgmt` folder, we can use `git config core.hooksPath 'tests/common/plugins/conditional_mark/hooks'` to specify the hook folder.
```

Comment:
Can the scripts be moved to folder like `sonic-mgmt/.hooks`? If they are in a deep subfolder, I don't think people will use them because the `git config` command line is a little bit long and hard to remember.

Reply:
done